pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -617,12 +619,13 @@ func (r *reconcilerBase) setupOrTeardown(ctx context.Context, syncObj client.Obj
 	// Else - the object is being deleted.
 
 	if controllerutil.ContainsFinalizer(syncObj, metadata.ReconcilerFinalizer) {
-		// The object is being deleted, but the reconciler finalizer is still running.
-		// Wait for the reconciler finalizer to complete.
-		r.logger(ctx).Info("Waiting for Reconciler Finalizer to finish")
```

Comment:
Should we keep the log and comment when reconciler finalizer isn't removed (L624)?

Reply:
done